packages/xo-web/src/common/intl/messages.js

```diff
@@ -1703,7 +1703,9 @@ const messages = {
   unknownSchedule: 'Unknown schedule',
   noDetachedBackups: 'No backups',
   noDuplicatedMacAddresses: 'No duplicated MAC addresses',
+  noOldObject: 'No snapshots older than 30 days',
   reason: 'Reason',
+  oldSnapshots: 'Snapshots older than 30 days',
```

Comment:
You have to be more explicit. Your feature displays snapshots older than 30 days for wich no schedule is enabled